doc/plotting/index.rst

```diff
@@ -459,4 +459,20 @@ Or you can save it to an html file::
 
 
 
-.. image:: ../images/plotly_markers_plot.png
\ No newline at end of file
+.. image:: ../images/plotly_markers_plot.png
+
+
+.. _interactive-stat-map-plotting:
+
+Interactive visualization of statistical map slices
+---------------------------------------------------
+
+:func:`view_stat_map`: open stat map in a Papaya viewer (https://github.com/rii-mango/Papaya)::
+
+    >>> from nilearn import plotting, datasets     # doctest: +SKIP
+    >>> img = datasets.fetch_localizer_button_task()['tmaps'][0]     # doctest: +SKIP
+    >>> view = plotting.view_stat_map(img, threshold=2, vmax=4)     # doctest: +SKIP
```

Comment:
Here I would separate two cases: viewing in an external browser window, or viewing in the notebook.